libs/sdk-ui-tests-e2e/cypress/tools/filterBar.ts

```diff
@@ -462,18 +475,38 @@ export class AttributeFilter {
      */
     configureLimitingMetricDependency(metricName: string | string[]) {
         this.selectConfiguration();
-
         const metrics = typeof metricName === "string" ? [metricName] : metricName;
+        const attributeFilterLimit = new AttributeFilterLimit();
```

Comment:
we created this method getAttributeFilterLimit() above, so I think we don't need to redefine it here. We can call directly getAttributeFilterLimit.addMetric().selectMetricItem(metricName);

Reply:
please also fix for these methods searchMetricDependency(), selectMetricDependency(), getNoDataMetricDependency()